doctrine.rst

```diff
@@ -812,18 +815,6 @@ control behavior:
         ): Response {
         }
 
-``exclude``
```

Comment:
there's no use case for `exclude` anymore IIUC, we could even deprecate it (but it's not strictly need either)